services/dynamic-sidecar/docker/entrypoint.sh

```diff
@@ -92,6 +92,8 @@ fi
 # directories are empty at this point
 # each individual subdirectory is a unique volume
 chown --verbose --recursive "$SC_USER_NAME":"$GROUPNAME" "${DY_VOLUMES}"
+# permssions rwxrwxr--
+chmod -R 774 "${DY_VOLUMES}"
```

Comment:
hmm I think you need better explanations of what is going on here? also could you please use verbose options?
How different is it from the usual permissions? why? are there implications?

Reply:
Default permissions are missing execute from group. When the service access files downloaded by the dynamic-sidecar it uses group permissions.